rotkehlchen/tests/api/test_metadata.py

```diff
@@ -0,0 +1,41 @@
+from typing import TYPE_CHECKING
+
+import requests
+
+from rotkehlchen.chain.ethereum.airdrops import AIRDROPS
+from rotkehlchen.chain.ethereum.defi.protocols import DEFI_PROTOCOLS
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+
+airdrops_expected_result = [
+    {
+        'identifier': identifier,
+        'name': airdrop.name,
+        'icon': airdrop.icon,
+    }
+    for identifier, airdrop in AIRDROPS.items()
+]
+
+defi_expected_result = [
+    {
+        'identifier': identifier,
+        'name': protocol.name,
+        'icon': protocol.icon,
+    }
+    for identifier, protocol in DEFI_PROTOCOLS.items()
+]
```

Comment:
create those inside the test otherwise they live as objects in memory as long as pytest is running

Reply:
alright